media/css/firefox/channel.scss

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/hero';
+@import '../../protocol/css/components/picto-card';
+@import '../protocol/components/sub-navigation';
+@import '../../protocol/css/components/newsletter-form';
+@import '../../protocol/css/components/call-out';
+@import '../../protocol/css/components/emphasis-box';
+
+//* -------------------------------------------------------------------------- */
+// Hero
+.mzp-c-hero.mzp-t-dark {
+    background-color: $color-off-black;
+}
+
+//* -------------------------------------------------------------------------- */
+// Picto Card
+
+.bugzilla .mzp-c-card-picto-content::before {
+    background: transparent url('/media/img/icons/warning.svg') no-repeat center;
+}
+
+//* -------------------------------------------------------------------------- */
+// Call-outs
+
+.mzp-c-call-out-compact {
+    background: transparent;
+}
+
+.mzp-l-notes {
+    margin: $layout-lg $layout-xs;
+}
+
+@media #{$mq-lg} {
+    .mzp-t-beta {
+        @include at2x('/media/protocol/img/logos/firefox/browser/beta/logo-md.png');
```

Comment:
You're no doubt already aware, but the logo sizing needs some work on larger viewports:

![image](https://user-images.githubusercontent.com/400117/70045729-79d10d80-15bc-11ea-8810-1584c015c873.png)